### PR TITLE
Enable root ssh on slem 6.0 firstrun

### PR DESCRIPTION
**Pull request by rfan1**
https://progress.opensuse.org/issues/154309

- Verification run: https://openqa.suse.de/tests/13373033

### CODE REVIEW

**Review by ilmanzo — APPROVED** [review]

File: tests/jeos/firstrun.pm [hunk 1]
```diff
@@ -239,6 +239,17 @@ sub run {
 
     # release console and reattach to be used again as serial output
     if (is_s390x && is_svirt) {
+        # enalbe root ssh login, see poo#154309
```
Comment (ricardobranco777) [inline tests/jeos/firstrun.pm:242]:
```suggestion
        # enalbe root ssh login, see poo#154309
```

Please fyx typo or remove the comment.

File: tests/jeos/firstrun.pm [hunk 2]
```diff
@@ -239,6 +239,17 @@ sub run {
...
+        if (is_alp || is_sle_micro('>=6.0')) {
+            record_info "enable root ssh login";
+            enter_cmd "root";    # login to serial console at first
+            wait_still_screen 1;
+            enter_cmd "$testapi::password";
+            wait_still_screen 1;
+            enter_cmd "mkdir -p /etc/ssh/sshd_config.d";
```
Comment (ricardobranco777) [inline tests/jeos/firstrun.pm:249]:
```suggestion
            enter_cmd "mkdir -p /etc/ssh/sshd_config.d";
```

Is this really needed?

**Review by ricardobranco777 — APPROVED** [review]
Couple of suggestions. Otherwise LGTM

File: tests/jeos/firstrun.pm (hunk 2 above)
Comment (rfan1) [reply tests/jeos/firstrun.pm:249]: Good question, let me check!

Comment (rfan1) [reply tests/jeos/firstrun.pm]: It is not needed, then remove this line

**Review by jlausuch — APPROVED** [review]

**Review by ricardobranco777 — APPROVED** [review]

File: tests/jeos/firstrun.pm [hunk 3]
```diff
@@ -239,6 +239,16 @@ sub run {
 
     # release console and reattach to be used again as serial output
     if (is_s390x && is_svirt) {
+        # enable root ssh login, see poo#154309
+        if (is_alp || is_sle_micro('>=6.0')) {
```
Comment (jlausuch) [inline tests/jeos/firstrun.pm:243]: at some point we would need to include `is_sle_micro('>=6.0')` as part of the `is_alp` condition. But let's be careful and have it in a new PR. I can take care of it.

**Pull request merged**